browser/src/map/Clipboard.js

```diff
@@ -1264,7 +1264,18 @@ L.Clipboard = L.Class.extend({
 	_warnCopyPaste: function() {
 		var id = 'copy_paste_warning';
 		this._map.uiManager.showYesNoButton(id + '-box', '', '', _('OK'), null, null, null, true);
+		this._warnCopyPasteImpl(id);
+	},
+
+	_warnCopyPasteImpl: function (id) {
 		var box = document.getElementById(id + '-box');
+
+		// TODO: do it JSDialog native...
+		if (!box) {
+			setTimeout(() => { this._warnCopyPasteImpl(id) }, 10);
+			return;
+		}
+
```

Comment:
Same comment as before, this should probably be a layouting task if it relies on elements that get added via layouting tasks. The pattern of "this thing isn't here now, just try again infinitely every 10ms" feels like a problem waiting to happen.

Reply:
That thing was converted from old "build from scratch with some library" to at least share our JSDialog window.
So now it creates window and then hacks it's inner content. It should be later moved to "native" JSDialog think which will not have that problem to solve (waiting for window).